src/impl_/pymodule.rs

```diff
@@ -147,7 +173,7 @@ mod tests {
             ModuleDef::new(
                 "test_module\0",
                 "some doc\0",
-                ModuleInitializer(|_, m| {
+                ModuleInitializer::GilRef(|_, m| {
```

Comment:
I also couldn't get `.into()` to work here, so I suspect there's just a problem with my `From` impls.